data/bib/dcallega.yaml

```diff
@@ -0,0 +1,19 @@
+references:
```

Comment:
The generated YAML for the new entry should go to the
`data/bib/iasl.yaml` file instead of this one. Perhaps the entry had
been added to the `static/bib/dcallega.bib` file first when this YAML
file has been regenerated.

Please regenerate the YAML files in the `data/bib` directory by
running the `scripts/mkbibdata` script from the root directory of
the repository.

Reply:
Did this

Reply:
Please remove this file (`data/bib/dcallega.yaml`) altogether.